pkg/util/proto/openapi.go

```diff
@@ -242,6 +242,21 @@ func (p *Primitive) GetName() string {
 	return fmt.Sprintf("%s (%s)", p.Type, p.Format)
 }
 
+// AnyValue is an arbitrary value (object, array or primitive)
```

Comment:
Can we figure out a better name? My initial understanding was that this was "AnyOf" (from openapi 3.0).